pkg/explain/printer_test.go

```diff
@@ -61,7 +61,7 @@ func Test_PrintFields(t *testing.T) {
       FeatureGates enables a set of custom feature gates. May only be used in conjunction with FeatureSet "CustomNoUpgrade". Features may be enabled or disabled by providing a true or false value for the feature gate. E.g. "featureGates": ["FeatureGate1=true", "FeatureGate2=false"].
 
     featureSet <string>
-      FeatureSet enables features that are not part of the default feature set. Valid values are "Default", "TechPreviewNoUpgrade" and "CustomNoUpgrade". When omitted, the "Default" feature set is used.
+      FeatureSet enables features that are not part of the default feature set. Valid values are "Default", "TechPreviewNoUpgrade", "DevPreviewNoUpgrade",  and "CustomNoUpgrade". When omitted, the "Default" feature set is used.
```

Comment:
```suggestion
      FeatureSet enables features that are not part of the default feature set. Valid values are "Default", "TechPreviewNoUpgrade", "DevPreviewNoUpgrade", and "CustomNoUpgrade". When omitted, the "Default" feature set is used.
```
@deads2k  there's an extra space here, causing the unit tests to fail. This test is the worst. We need to rework it.